css/style.css

```diff
@@ -0,0 +1,710 @@
+body {
+  font-family: nyt-franklin, helvetica, arial, sans-serif;
+  font-size: 14.5px;
+  margin-left: 0;
+  margin-right: 0;
+}
+
+a {
+  color: rgb(50, 104, 145);
+}
+
+h1 h3 {
+  color: #121212;
+  font-family: nyt-cheltenham, georgia, 'times new roman', times, serif;
+  font-weight: 700;
+  font-style: italic;
+  font-size: 2.1375rem;
+  line-height: 2.95rem;
+  margin: 0 20px 1rem;
+  position: relative;
+  width: calc(100% - 40px);
+  max-width: 600px;
+}
+
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  font-weight: 700;
+}
+
+.navbar-right-list {
+  float: right;
+  margin-right: 4px;
+}
+
+li {
+  float: left;
+}
+
+li img {
+  position: absolute;
+  left: 42%;
+  margin-top: 4px;
+}
+
+li a {
+  display: block;
+  text-align: center;
+  padding: 8px 16px 14px 16px;
+  text-decoration: none;
+  color: black;
+}
+
+.all-footer {
+  text-align: left;
+}
+
+.article-name {
+  display: flex;
+  flex-flow: column;
+  justify-content: flex-start;
+}
+
+.logo-link a {
+  float: right;
+}
+
+.article-content-on-left {
+  flex: 55%;
+  width: 600px;
+  padding: 20px;
+  margin: 10px;
+
+  /* border-style: dotted; */
```

Comment:
Please remove commented code. 
![Screen Shot 2020-06-13 at 1 50 24 PM](https://user-images.githubusercontent.com/48804258/84568019-ea7f9080-ad7c-11ea-8745-7aac6be26df7.png)

Reply:
Removed
Thanks for the review and helps me to guide